src/components/expanded-state/UniqueTokenExpandedState.js

```diff
@@ -54,6 +64,50 @@ const UniqueTokenExpandedState = ({ asset }) => {
   const { height: screenHeight } = useDimensions();
   const { colors, isDarkMode } = useTheme();
 
+  const { supportsAudio } = useUniqueToken(asset);
+  const {
+    playAsset,
+    currentlyPlayingAsset,
+    isPlayingAssetPaused,
+    fadeTo,
+    currentSound,
+    stopPlayingAsset,
+  } = useAudio();
+
+  const assetIsPlayingAudio =
+    !!currentlyPlayingAsset &&
+    currentlyPlayingAsset.uniqueId === asset.uniqueId;
+
+  // TODO: if playing etc
+  const handlePressAudio = React.useCallback(async () => {
+    try {
+      if (assetIsPlayingAudio) {
+        const shouldPause = () => currentSound.pause();
+        const shouldPlay = () => currentSound.play();
+        return isPlayingAssetPaused ? shouldPlay() : shouldPause();
+      }
+      return playAsset(asset);
+    } catch (e) {
+      logger.error(e);
+    }
+  }, [
+    asset,
+    currentSound,
+    playAsset,
+    assetIsPlayingAudio,
+    isPlayingAssetPaused,
+  ]);
+
+  React.useEffect(() => {
+    EXPERIMENTAL_AUDIO_PLAYER !== 'true' && playAsset(asset);
```

Comment:
is it experimental? I think those changes are super easily revertable/easy to disable that we can just merge this without flags. We were using those flags for features that are complex and hard to remove if needed